src/Fixer/Basic/Psr0Fixer.php

```diff
@@ -158,6 +158,7 @@ protected function createConfigurationDefinition()
         return new FixerConfigurationResolver([
             (new FixerOptionBuilder('dir', 'The directory where the project code is placed.'))
                 ->setAllowedTypes(['string'])
+                ->setDefault('')
```

Comment:
So, should we remove [this example](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/c606b75a87baf017178b95f93406942f7816277f/src/Fixer/Basic/Psr0Fixer.php#L39-L45) from the fixer?